tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/questionnaire_response_id_map_test.py

```diff
@@ -44,12 +45,13 @@ def setUpClass(cls):
         cls.fq_table_names = [
             f'{project_id}.{dataset_id}.{OBSERVATION}',
             f'{project_id}.{dataset_id}.{SURVEY_CONDUCT}',
-            f'{project_id}.{dataset_id}.{DEID_QUESTIONNAIRE_RESPONSE_MAP}'
+            f'{project_id}.{dataset_id}.{DEID_QUESTIONNAIRE_RESPONSE_MAP}',
+            f'{project_id}.{dataset_id}.{SURVEY_CONDUCT}{EXT_SUFFIX}'
         ]
 
         sb_table_name = cls.rule_instance.sandbox_table_for(SURVEY_CONDUCT)
         cls.fq_sandbox_table_names = [
-            f'{project_id}.{cls.sandbox_id}.{sb_table_name}'
+            # f'{project_id}.{cls.sandbox_id}.{sb_table_name}'
```

Comment:
Can we either remove this commented line or add text around it as to why it's not needed?

Reply:
This was in case you wanted to talk about removing the sandbox and delete queries.  Maybe we can talk about it tomorrow.